interfaces/builtin/modem_manager.go

```diff
@@ -1358,7 +1358,12 @@ func (iface *modemManagerInterface) AppArmorConnectedPlug(spec *apparmor.Specifi
 }
 
 func (iface *modemManagerInterface) DBusConnectedPlug(spec *dbus.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
-	spec.AddSnippet(modemManagerConnectedPlugDBus)
+	// Don't create default policy on classic otherwise it will likely
+	// conflict with any existing policy for ModemManager from the
+	// distro itself
+	if !release.OnClassic {
```

Comment:
I looked at network manager, and perhaps the plug rule is overly restrictive. Would it make sense if the deny rules for `context="default"` was rather a part of the permanent slot dbus snippet like it is for NM?

Reply:
Ah meh, the slot is provided by system, so meh, that won't work AFAICT.

Reply:
If we are on a classic system, we have the D-Bus policy in `/etc/dbus-1/system.d/org.freedesktop.ModemManager1.conf` to rely on. There really isn't any reason for snapd to be installing policy unless a modem-manager snap is installed.